Allow thumb flag to be passed to make (e.g. -mthumb)
By Yauheni Kaliuta <yauheni.kaliuta@nokia.com>

Ported to BusyBox 1.18.4 by Dennis Groenen <dennis_groenen@hotmail.com> - 2011-04-29  
---

--- a/Makefile.flags
+++ b/Makefile.flags
@@ -51,7 +51,7 @@ CFLAGS += $(call cc-option,-fno-builtin-
 # of branch probabilities (hopefully makes bloatcheck more stable):
 CFLAGS += $(call cc-option,-fno-guess-branch-probability,)
 CFLAGS += $(call cc-option,-funsigned-char -static-libgcc,)
-CFLAGS += $(call cc-option,-falign-functions=1 -falign-jumps=1 -falign-labels=1 -falign-loops=1,)
+CFLAGS += $(call cc-option,-falign-functions=1 -falign-jumps=1 -falign-labels=1 -falign-loops=1 $(THUMB),)
 
 # FIXME: These warnings are at least partially to be concerned about and should
 # be fixed..
